<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderDaig</class>
 <widget class="QDialog" name="ReaderDaig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>472</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select TEC Files to Read</string>
  </property>
  <widget class="QListWidget" name="fileListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addFileBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="140">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="140">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="140">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="toolTip">
    <string>選擇加入讀取清單中的TEC檔案</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFileBtn">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="140">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="140">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="140">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="toolTip">
    <string>移除所選的TEC檔案</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTECBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelLoadBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>280</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projFolderEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectProjFolder">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>34</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria</family>
    </font>
   </property>
   <property name="toolTip">
    <string>設定專案資料夾</string>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please Select a Project Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="geoRefBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>23</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>設定參考座標系</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="chageAllBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>all</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="changeOneBtn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>7</y>
      <width>98</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>single file</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="attributeList">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="callSettingsBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>23</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>轉Raster圖層設定</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
